src/services/redisSessionStorage.ts

```diff
@@ -953,9 +965,11 @@ export default class RedisSessionStorage extends RedisStorageUtils
                   ) &&
                   !(p.prefix === fimsTokenPrefix && p.value === user.fims_token)
               ),
-              (_1, { prefix, value }) => `${prefix}${value}`
+              R.collect(Ord)((_1, { prefix, value }) => `${prefix}${value}`)
```

Comment:
We are ordering the results here.
In other place we use the deprecated `R.collect` without the `Ord` constraint.
```suggestion
              R.collect((_1, { prefix, value }) => `${prefix}${value}`)
```
Maybe this is an improvement but the output is different.